docs/guide/dev/deep_dive/errors/README.md

```diff
@@ -23,6 +23,16 @@ This article contains the list of CKEditor error codes and their explanation. Th
 * Description: Incorrect {@linkapi CKEDITOR.config#autoEmbed_widget } value. No widget definition found.
 * Additional data: None.
 
+## cloudservices-no-url
+* Location: `plugins/cloudeservices/plugin.js`
+* Description: Upload url used by {@linkapi CKEDITOR.config.cloudServices} is not set. An url can be provided by configuration {@linkapi CKEDITOR.config#url} property or passed to [Cloud Services](https://ckeditor.com/ckeditor-cloud-services/) plugin {@linkapi CKEDITOR.plugins.cloudeServices.cloudServicesLoader#upload} method.
```

Comment:
Link to the addon will be `https://ckeditor.com/cke4/addon/cloudservices`. You're talking about the plugin here, so you need to link the plugin not the whole CS product itself.